nautobot/core/views/mixins.py

```diff
@@ -732,7 +732,11 @@ def get_template_name(self):
                     except TemplateDoesNotExist:
                         # Try a different detail view template format
                         template_name = f"{app_label}/{model_opts.model_name}.html"
-                        select_template([template_name])
+                        try:
+                            select_template([template_name])
+                        except TemplateDoesNotExist:
+                            # Catch-all fallback to just object_retrieve.html
+                            template_name = "generic/object_retrieve.html"
```

Comment:
BTW, this is needed for custom templates that don't use the proper CRUD "action" words (e.g., `cable_connect.html`, `cable_trace.html`, etc).